addons/website/static/src/xml/website_widget.xml

```diff
@@ -2,15 +2,19 @@
 
 <templates xml:space="preserve">
 
-    <t t-name="website.iframeWidget">
-        <iframe
-            t-if="url"
-            height="100%"
-            width="100%"
-            frameBorder="0"
-            t-att-src="url"
-            class='d-block'
-        ></iframe>
+    <t t-name="website.iframeWidget" owl="1">
+        <div t-if="props.value" t-att-class="this.state.classes">
+            <iframe
+                frameBorder="0"
+                class="d-block"
+                height="100%"
+                width="100%"
+                t-att-src="props.value"/>
+            <img alt="phone"
+                 class="img_mobile"
+                 style="display: none;"
+                 src="/website/static/src/img/phone.png"/>
```

Comment:
Is this needed to have it in display: none and show it via the class? Can't this just be not rendered at all? Maybe it flickers if we do that?